README.md

```diff
@@ -203,12 +231,67 @@ One good way to automate is using CloudFormation, using this example snippet:
 }
 ```
 
-#### Elastic Beanstalk Installation ####
+#### For Elastic Beanstalk ####
 
 * In order to add tags to existing Elastic Beanstalk environment, you will
   need to rebuild the environment with the spot-enabled tag. Follow this
   [guide](http://www.boringgeek.com/add-or-update-tags-on-existing-elastic-beanstalk-environments)
 
+### Configuration of autospotting ###
+#### Default configuration ####
```

Comment:
Yes, I thought about that, but that would likely require a rebuild when changing the configuration, since the config file would need to embedded into the function code, as opposed to the CloudFormation mechanism which only requires some parameter changes.

The current implementation in #57 is based on information passed by the CloudWatch Event, but I just realized we could use the relatively new Lambda environment variables to achieve the same functionality. I'll give that a try since it may be cleaner than the CloudWatch Event approach and would also work for local execution.